docs/index.md

```diff
@@ -25,12 +23,12 @@ Taipy is a complete tool that empowers individuals and enterprises to develop ap
 **Back-End Functionalities:**
 
 - Taipy provides extensive support for establishing resilient pipelines capable of managing diverse scenarios.
-- The process of modeling Directed Acyclic Graphs (DAGs) is simplified through data flow functionality.
-- Enhancing the overall performance of Taipy applications, the data caching feature is integrated.
-- Facilitating a registry of pipeline executions.
+- The process of modeling Directed Acyclic Graphs (DAGs) is simplified through Taipy Studio.
+- Enhancing the overall performance of Taipy applications, data caching, parallel executions of tasks, data source scoping are all integrated.
+- Providing a registry of pipeline executions.
 - Enabling pipeline versioning.
 - Taipy equips users with the ability to monitor and assess the performance of their applications using the KPI tracking tool.
```

Comment:
I don't know what it is referring to?

Reply:
There is no KPI tracking tool?

Reply:
Not really. However, we can use the visual elements to monitor some data nodes. I proposed a change in the following line to express that idea.